clients/unshaded/src/main/java/tachyon/client/file/FileInStream.java

```diff
@@ -126,7 +126,7 @@ public int read() throws IOException {
       try {
         mCurrentCacheStream.write(data);
       } catch (IOException ioe) {
-        LOG.warn("Block of ID {} could not be cached into Tachyon. Exception: {}",
+        LOG.warn("The blockId {} could not be cached into Tachyon. Exception: {}",
```

Comment:
`LOG.warn("abc", exception);` will print "abc" and then print the exception with its stack trace on the next lines. This is ideal for serious warnings and errors like this one.